lib/material/internal/material_tabs.flow

```diff
@@ -221,11 +224,19 @@ MDynamicTabs2T(manager : MaterialManager, parent : MFocusGroup, m : MDynamicTabs
 									\focus ->
 										TGroup2(sz, TShowLazy(fOr(focus.focused, focus.down), \ -> TRectangle([MFill(MGrey(500)), FillOpacity(0.3)], sz)))
 										|> (\t2 -> TConstruct([
-												\ -> fconnect(
+											\ -> fconnect(
 													fmaxA([feq(m.selected, i), focus.down, focus.focused, focus.hover], false),
 													isActive
 												)
-											], t2)),
+											], 
+											eitherMap(tabBarStyleOnSelectedM,
```

Comment:
Also, better to move into the group between `sz` and `TShowLazy(fOr(focus.focused, focus.down), ...)` to keep focusing overlay visible